src/main/java/org/mule/module/apikit/validation/body/form/transformation/MultipartWithDefaults.java

```diff
@@ -28,7 +28,9 @@ public MultipartWithDefaults(HttpEntity multipartFormEntity) {
   @Override
   public InputStream content() throws InvalidFormParameterException {
     try {
-      return multipartFormEntity.getContent();
+      ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
+      multipartFormEntity.writeTo(byteArrayOutputStream);
+      return new ByteArrayInputStream(byteArrayOutputStream.toByteArray());
```

Comment:
Wouldn't this result in `OutOfMemoryError` for really big payloads?

`org.apache.http.ContentTooLongException` makes sense as an exception (for a bigger buffer size, obviously).

Reply:
Yes, that's the drawback of manipulating defaults.
The exception thrown is from the extension (with APIKIT:BAD_REQUEST type) which wraps the underling one.